pkg/features/kube_features.go

```diff
@@ -232,7 +232,7 @@ var defaultVersionedFeatureGates = map[featuregate.Feature]featuregate.Versioned
 		{Version: version.MustParse("0.10"), Default: false, PreRelease: featuregate.Alpha},
 	},
 	TASLeastAllocated: {
-		{Version: version.MustParse("0.11"), Default: false, PreRelease: featuregate.Deprecated},
+		{Version: version.MustParse("0.11"), Default: false, PreRelease: featuregate.Alpha},
```

Comment:
Actually, we recommended using `Deprecated` instead of `Alpha` since we strongly think that this should be removed in the future. The only concern is mitigating pain for the existing TAS users.

@mimowo Do we really want to say this as `Alpha`?

Reply:
Yes, I think deprecated better reflects the intention. Alpha to me is feature gate that is planned to be stable one day. This is not the case here. However, if it is causing practical issues, then I'm ok to call it Alpha and just say "Deprecated" in a comment. 

So, I would like to understand the practical problems - what this means " This was adding noise to unit tests.", @gabesaba ?

Reply:
I didn't realize this was intentional. I just saw a bunch of warning messages referencing the use of this deprecated flag. This would display when running pkg/cache tests, and one of the tests was failing (thereby increasing verbosity)

I am okay with this PR being closed, then.